driver-adapters/neon-http-lambda-basic/index.js

```diff
@@ -26,6 +26,18 @@ exports.handler = async () => {
         name: true,
       },
     }),
+    create2: await prisma.user.create({
```

Comment:
```suggestion
    // Since `createMany` does not work we add a second create 
    // the record is used for `findUniqueOrThrow(...)` below
    create2: await prisma.user.create({
```